synapse/storage/schema/full_schemas/54/zzz-inserts.sql

```diff
@@ -0,0 +1,12 @@
+INSERT INTO user_directory_stream_pos (stream_id) VALUES (null);
+
+INSERT INTO appservice_stream_position (stream_ordering)
+    SELECT COALESCE(MAX(stream_ordering), 0) FROM events;
```

Comment:
erm; surely this (and device_inbox) will be empty, so this is a no-op?

Reply:
true, i just copy pasted so I couldn't mess it up :)